src/control-plane/backend/lambda/api/service/quicksight/reporting-utils.ts

```diff
@@ -660,11 +700,11 @@ export function sleep(ms: number) {
 export function getQuickSightUnitFromTimeUnit(timeUnit: string) : string {
 
   let unit = 'DAY';
-  if (timeUnit == 'WK') {
+  if (timeUnit == ExploreRelativeTimeUnit.WK) {
```

Comment:
why not directly using `WEEK` as enum `ExploreRelativeTimeUnit` value?

Reply:
fixed in #178